lib/workers/repository/update/branch/index.spec.ts

```diff
@@ -1956,5 +1956,59 @@ describe('workers/repository/update/branch/index', () => {
         result: 'done',
       });
     });
+
+    it('continues branch, skips automerge if there are artifact errors', async () => {
+      jest.spyOn(getUpdated, 'getUpdatedPackageFiles').mockResolvedValueOnce({
+        updatedPackageFiles: [{}],
+        artifactErrors: [{}],
+      } as PackageFilesResult);
+      npmPostExtract.getAdditionalFiles.mockResolvedValueOnce({
+        artifactErrors: [],
+        updatedArtifacts: [],
+      });
+      git.branchExists.mockReturnValue(true);
```

Comment:
why not once here? not using once can cause side effects to other tests.

Reply:
just copy paste mistake xD